test/test_torch.py

```diff
@@ -12169,6 +12169,17 @@ def test_comparison_ops(self):
         for idx in iter_indices(x):
             self.assertEqual(x[idx] >= y[idx], ge[idx] == 1)
 
+    def test_comparison_ops_with_type_promotion(self):
```

Comment:
might want to put this in test_type_promotion.py

Reply:
let's also make sure we've covered the specific class of cases described by the user-reported issue:
https://github.com/pytorch/pytorch/issues/22301

Reply:
Moved and added special test case

Reply:
@ifedan is there still a test case for this?

Reply:
This should be fixed with type promotion. For now it works the same as previously